labs/8.md

```diff
@@ -1,9 +1,103 @@
-## Conclusion
+# Convert a Namespace to a Federated Namespace
+Applications within OpenShift may already be deployed in an unfederated way. The `kubefedctl` tool will allow for a namespace and the objects within that namespace to become federated objects. This lab will deploy an application with a deployment and service and then use kubefedctl to federate the application and have it running on all three clusters. 
 
-Thanks for attending this hands on lab. By now you have completed the following activities.
+## Create a Namespace
+To start with we will create a namespace for the *reverse-words* application to be deployed.
 
-* Explored the environment
-* Deployed KubeFed
-* Flattened out your configuration files
-* Deployed 2 applications
-* Etc...
+~~~sh
+oc create ns --context cluster1 reverse-words
+oc project --context cluster1 reverse-words
```

Comment:
I personally try to avoid the use of oc project as it cannot be replaced by any kubectl command, instead, I rather use "-n <namespace>". Feel free to modify it or leave it as it's now.

Reply:
You can try this:

```
(kubectl|oc) config set-context reverse-words --cluster cluster1 --user cluster1 --namespace reverse-words
(kubectl|oc) config use-context reverse-words
```

Reply:
Yes, that will work as well. I was referring to the fact that kubectl doesn't have "kubectl project" command (which basically does what you mentioned). 

I'm okay with any approach you want to follow.